icubam/db/test_wipe.py

```diff
@@ -0,0 +1,33 @@
+from absl.testing import absltest
+from itertools import repeat
+from sqlalchemy import create_engine
+from icubam.db.fake import populate_store_fake
+from icubam.db.wipe import wipe_db
+from icubam.db.store import Store, StoreFactory
+
+class WipeTest(absltest.TestCase):
+    def setUp(self):
```

Comment:
you are using 4 spaces. please stick to 2

Reply:
Will do, thanks!